### PR TITLE
docs(issue-templates): add issue templates to repository

### LINKED ISSUE #305: Create issue templates
When I create a new `octokit/*` repository, I use https://github.com/octokit/create-octokit-project.js, which among other things creates issue templates. The templates that it creates are here: https://github.com/octokit/create-octokit-project.js/blob/81ddfae910fcf7bf3e81407360db0db80ab05d49/lib/create-issue-templates.js

I think it would help to create issue templates in this repository. But then also create one special issue template for the specific case when Types for an event payload are incorrect / incomplete. We have seen a lot of these lately, and we could put in some guidance into this specific template, and also ask for an example event payload that either we can just add to https://github.com/octokit/webhooks ourselves, or even better ask the user to send a PR

### PR DESCRIPTION
**Pull request by oscard0m**
## 📝 Summary

Add ISSUE templates to @octokit/webhooks.js repository:
* The ones generated from https://github.com/octokit/create-octokit-project.js
* A new template for *Types for an event payload are incorrect / incomplete*

## ⛱ Motivation and Context

Fix #305 

-----
[View rendered .github/ISSUE_TEMPLATE/01_help.md](https://github.com/dominguezcelada/webhooks.js/blob/add-issue-templates-to-repository/.github/ISSUE_TEMPLATE/01_help.md)
[View rendered .github/ISSUE_TEMPLATE/02_bug.md](https://github.com/dominguezcelada/webhooks.js/blob/add-issue-templates-to-repository/.github/ISSUE_TEMPLATE/02_bug.md)
[View rendered .github/ISSUE_TEMPLATE/03_feature_request.md](https://github.com/dominguezcelada/webhooks.js/blob/add-issue-templates-to-repository/.github/ISSUE_TEMPLATE/03_feature_request.md)
[View rendered .github/ISSUE_TEMPLATE/04_thanks.md](https://github.com/dominguezcelada/webhooks.js/blob/add-issue-templates-to-repository/.github/ISSUE_TEMPLATE/04_thanks.md)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/05_type_pyaloads_incorrect_or_incomplete.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+<!-- Please share which payload/s is/are failing -->
+
+**Example of a Payload fixed**
+
+<!-- Share a snippet proposal of a working payload example if possible -->
+<!-- you can get examples in https://github.com/octokit/webhooks/tree/master/payload-examples/api.github.com -->
+
+**Linked issue in @octokit/webhooks**
+
+<!-- Create a linked issue to this issue on @octokit/webhooks, once is fixed this package will get the fix in next update -->
```
Comment (oscard0m) [inline .github/ISSUE_TEMPLATE/05_type_pyaloads_incorrect_or_incomplete.md]: If you agree on this point, I can create an issue on @octokit/webhooks adding an `ISSUE TEMPLATE`

@gr2m @wolfy1339

Comment (wolfy1339) [inline .github/ISSUE_TEMPLATE/05_type_pyaloads_incorrect_or_incomplete.md]: Not sure completely about the wording, it doesn't ring right to me. I'm not entirely sure how to rephrase it, but this is what I think is better.
Also added a link to the new issue page
```suggestion
<!-- Create an issue on @octokit/webhooks (https://github.com/octokit/webhooks/issues/new) and add a link back to this one. Once it is fixed there, this package will get the fix during the next update -->
```

**Review by wolfy1339 — APPROVED** [review]

**Review by gr2m — COMMENTED** [review]
This is great, thank you both for creating an reviewing it!

I wonder if we should move the payload types bug report above the other bug report? https://github.com/octokit/webhooks.js/issues/new/choose

I fear folks might see bug report and just go with and missing the 5th option at the end

**Comment by oscard0m** [comment]
> This is great, thank you both for creating an reviewing it!
> 
> I wonder if we should move the payload types bug report above the other bug report? [octokit/webhooks.js/issues/new/choose](https://github.com/octokit/webhooks.js/issues/new/choose)
> 
> I fear folks might see bug report and just go with and missing the 5th option at the end

Could be confusing, you are right. Creating issue+PR to re-sort it ;)

**Pull request merged**